content/en/examples/service/networking/tls-example-ingress.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: tls-example-ingress
+spec:
+  tls:
+  - hosts:
+      - sslexample.foo.com
+    secretName: testsecret-tls
+  rules:
+  - host: sslexample.foo.com
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: service1
+            port:
+              number: 80
```

```suggestion
  - host: https-example.foo.com
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: service1
            port:
              number: 443
```

I prefer not to write “SSL” in formal docs. SSL is so last century :wink: (TLS 1.0 came out in 1999).